packages/arb-validator/cmdhelper/cmdhelper.go

```diff
@@ -21,139 +21,108 @@ import (
 	"flag"
 	"fmt"
 	"log"
-	"math"
-	"math/big"
 	"os"
 	"path/filepath"
-	"strings"
-	"syscall"
 	"time"
 
+	"github.com/offchainlabs/arbitrum/packages/arb-validator-core/utils"
+
 	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/gorilla/rpc"
+	"github.com/gorilla/rpc/json"
 
 	"github.com/offchainlabs/arbitrum/packages/arb-util/common"
 	"github.com/offchainlabs/arbitrum/packages/arb-validator-core/arbbridge"
 	"github.com/offchainlabs/arbitrum/packages/arb-validator-core/ethbridge"
 	"github.com/offchainlabs/arbitrum/packages/arb-validator/rollup"
 	"github.com/offchainlabs/arbitrum/packages/arb-validator/rollupmanager"
 	"github.com/offchainlabs/arbitrum/packages/arb-validator/rollupvalidator"
-
-	"golang.org/x/crypto/ssh/terminal"
-
-	"github.com/ethereum/go-ethereum/accounts"
-	"github.com/ethereum/go-ethereum/accounts/abi/bind"
-	"github.com/ethereum/go-ethereum/accounts/keystore"
 )
 
-func GetKeystore(validatorFolder string, pass *string, flags *flag.FlagSet) (*bind.TransactOpts, error) {
-	ks := keystore.NewKeyStore(filepath.Join(validatorFolder, "wallets"), keystore.StandardScryptN, keystore.StandardScryptP)
-
-	found := false
-	flags.Visit(func(f *flag.Flag) {
-		if f.Name == "password" {
-			found = true
-		}
-	})
-
-	var passphrase string
-	if !found {
-		if len(ks.Accounts()) == 0 {
-			fmt.Print("Enter new account password: ")
-		} else {
-			fmt.Print("Enter account password: ")
-		}
-
-		bytePassword, err := terminal.ReadPassword(int(syscall.Stdin))
-		if err != nil {
-			return nil, err
-		}
-		passphrase = string(bytePassword)
-
-		passphrase = strings.TrimSpace(passphrase)
-	} else {
-		passphrase = *pass
-	}
-
-	var account accounts.Account
-	if len(ks.Accounts()) == 0 {
-		var err error
-		account, err = ks.NewAccount(passphrase)
-		if err != nil {
-			return nil, err
-		}
-	} else {
-		account = ks.Accounts()[0]
-	}
-	err := ks.Unlock(account, passphrase)
-	if err != nil {
-		return nil, err
-	}
-	auth, err := bind.NewKeyStoreTransactor(ks, account)
-	if err != nil {
-		return nil, err
-	}
-	return auth, nil
-}
-
-func ValidateRollupChain(execName string, managerCreationFunc func(rollupAddress common.Address, client arbbridge.ArbAuthClient, contractFile string, dbPath string) (*rollupmanager.Manager, error)) error {
+func ValidateRollupChain(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported function ValidateRollupChain should have comment or be unexported](https://app.codacy.com/gh/OffchainLabs/arbitrum/pullRequest?prid=5428340)